### DISCUSSION TITLE
What are ‘nodes’?

### DISCUSSION THREAD

**Discussion by SHogenboom** [Erudika/scoold#385] [Q&A] [answered]
In the pricing for Scoold Cloud / Scoold Pro Hosting the only difference between the priceplans appears to be the number of ‘nodes’… What does that correspond to?

**Comment by MarcMechi98** [comment, marked as answer]
Hi! The term "nodes" refers to the number of instances or servers that are included in the pricing plans. Each node represents a separate server instance that runs Scoold, and having multiple nodes allows you to scale your application for better performance, reliability, and handling higher traffic loads.
